src/routes/routes.js

```diff
@@ -13,7 +12,7 @@ export default () => (
   <BrowserRouter>
     <Switch>
       <Route path="/scouting-form" exact component={ScoutingForm}/>
-      <Route path="/" exact component={HomePage}/>
+      <Route path="/" exact component={ScoutingForm}/>
```

Comment:
change the component to redirect on '/scouting-form'

Reply:
you mean component = <redirect path="/scouting-form">?

Reply:
<Redirect path="/scouting-form"/>